cdap-common/src/main/java/co/cask/cdap/common/conf/Configuration.java

```diff
@@ -540,6 +540,16 @@ public String getRaw(String name) {
   }
 
   /**
+   * Get all key-value kafka config pairs
+   * @return hashmap of <code>name</code> property and its value
+   */
+  public Map<String, String> getKafkaConfigs() {
+    //use getValByRegex to get all config keys that start with "kafka."
+    String kafkaRegex = "^(kafka.*)$";
```

Comment:
I would prefer the prefix as `kafka.server.*`, since there can be other configurations related to Kafka that we need but not related to kafka server.